superset/assets/src/components/FilterableTable/FilterableTableStyles.css

```diff
@@ -43,6 +43,11 @@
   border-right: 1px solid #ccc;
   align-self: center;
   padding: 12px;
+  font-family: monospace;
```

Comment:
I wonder if we could have a more specific font here first (falling back to `monospace` if not available), so the results look better? At least in the screenshot you posted the monospaced font seems harder to read than before.

Reply:
I tested with `'Roboto Mono', monospace` and it looks somewhat better IMHO.

Reply:
We'd need to include `roboto mono` with the website then, not sure if it's worth adding a dependency there on a font. Let's see what someone more design minded thinks

Reply:
Not necessarily, it would fall back to `monospace` if `Roboto Mono` is not installed.

GitHub uses `SFMono-Regular,Consolas,Liberation Mono,Menlo,Courier,monospace;`, eg.

Reply:
Different fonts would have different character widths, which would make this even harder to implement though..